tests/contrib/openstack/test_os_contexts.py

```diff
@@ -4455,8 +4459,8 @@ def test_resolve_pci_from_mapping_config(self):
 }
 
 NUMA_CORES_MULTI = {
-    '0': [0, 1, 2, 3],
-    '1': [4, 5, 6, 7]
+    '0': [0, 2, 4, 6],
+    '1': [1, 3, 5, 7]
```

Comment:
I think the CPU list per NUMA in the test data was incorrect. Here is an example of CPU list with 2 NUMA nodes:
# numactl -H
available: 2 nodes (0-1)
node 0 cpus: 0 2 4 6 8 10 12 14
node 1 cpus: 1 3 5 7 9 11 13 15